packages/experimental/Button/src/ButtonTokens.ts

```diff
@@ -1,139 +1,65 @@
+      hasContent: {
+        hasIcon: {
+          spacingIconContent: globalTokens.spacing.mNudge,
+        },
       },
-      hovered: {
-        backgroundColor: t.colors.brandedHoveredBackground,
-        color: t.colors.brandedHoveredContent,
-        borderColor: t.colors.brandedHoveredBorder,
-        iconColor: t.colors.brandedHoveredIcon,
-      },
-      pressed: {
-        backgroundColor: t.colors.brandedPressedBackground,
-        color: t.colors.brandedPressedContent,
-        borderColor: t.colors.brandedPressedBorder,
-        iconColor: t.colors.brandedPressedIcon,
-      },
-      focused: {
-        backgroundColor: t.colors.brandedFocusedBackground,
-        color: t.colors.brandedFocusedContent,
-        borderColor: t.colors.brandedFocusedBorder,
-        iconColor: t.colors.brandedFocusedIcon,
-      },
-    },
-    disabled: {
-      backgroundColor: t.colors.defaultDisabledBackground,
-      color: t.colors.defaultDisabledContent,
-      borderColor: t.colors.defaultDisabledBorder,
-      iconColor: t.colors.defaultDisabledIcon,
-    },
-    hovered: {
-      backgroundColor: t.colors.defaultHoveredBackground,
-      color: t.colors.defaultHoveredContent,
-      borderColor: t.colors.defaultHoveredBorder,
-      iconColor: t.colors.defaultHoveredIcon,
-    },
-    pressed: {
-      backgroundColor: t.colors.defaultPressedBackground,
-      color: t.colors.defaultPressedContent,
-      borderColor: t.colors.defaultPressedBorder,
-      iconColor: t.colors.defaultPressedIcon,
-    },
-    focused: {
-      backgroundColor: t.colors.defaultFocusedBackground,
-      color: t.colors.defaultFocusedContent,
-      borderColor: t.colors.defaultFocusedBorder,
-      icon: t.colors.defaultFocusedIcon,
     },
-    primary: {
-      backgroundColor: t.colors.brandedBackground,
-      color: t.colors.brandedContent,
-      borderColor: t.colors.brandedBorder,
-      iconColor: t.colors.brandedIcon,
-      disabled: {
-        backgroundColor: t.colors.brandedDisabledBackground,
-        color: t.colors.brandedDisabledContent,
-        borderColor: t.colors.brandedDisabledBorder,
-        iconColor: t.colors.brandedDisabledIcon,
-      },
-      hovered: {
-        backgroundColor: t.colors.brandedHoveredBackground,
-        color: t.colors.brandedHoveredContent,
-        borderColor: t.colors.brandedHoveredBorder,
-        iconColor: t.colors.brandedHoveredIcon,
-      },
-      pressed: {
-        backgroundColor: t.colors.brandedPressedBackground,
-        color: t.colors.brandedPressedContent,
-        borderColor: t.colors.brandedPressedBorder,
-        iconColor: t.colors.brandedPressedIcon,
-      },
-      focused: {
-        backgroundColor: t.colors.brandedFocusedBackground,
-        color: t.colors.brandedFocusedContent,
-        borderColor: t.colors.brandedFocusedBorder,
-        iconColor: t.colors.brandedFocusedIcon,
+    medium: {
+      padding: globalTokens.spacing.sNudge - 1,
```

Comment:
- 1 here is to adjust for the border taking up a pixel

Reply:
Is the border taking up a pixel only a win32 thing? AKA, can we assume this for other platforms?
Also should it be "globalTokens.stroke.width.thin" instead of just 1, in case the border width is not always 1?

Reply:
@Saadnajmi I know that UWP operates the same way but yeah, not sure about other platforms. That will be a point where we could get some per-platform styling. I can preemptively add a function that doesn't do anything for android or Apple.

And yeah, you're right, we should be using the value of the constant.

Reply:
I tried making borderRadius 10 on the apple theme for macOS, and all the buttons got bigger, so I guess this may be an actual platform difference :/ 
Good to keep in mind for future when we use the token pipeline tokens for experimental button on macOS and others!

Reply:
I extracted the constant out into a function which can give different values based on platform, but this file can be shared between the platforms. Not sure if this is the best way of handling this, but for now since is the main difference, I'll start with this and see if we need to adjust later.